.travis.yml

```diff
@@ -32,13 +32,13 @@ before_script:
   - pip install -r requirements.txt
 script:
   # run coverage on files testing the app module
-  - coverage run --source app -m unittest discover
-  - cp .coverage* ../
+  - coverage run --branch --omit="*/virtualenv/*","*.html" -m unittest discover
```

Comment:
I think coverage automatically ignores non-python files, so you don't need to ignore html files.

Reply:
I tried that, but it broke on Pageserve when running Coveralls. It tried to run the HTML then broke on line 1 saying it wasn't Python. Pageserve still passed the Travis build for some reason, but it wasn't able to report coverage to Coveralls. Really annoying, but this fixes it.

Reply:
I haven't been able to find out why that happens, so that seems like the best way to handle it.